server/services/store/sqlstore/migrations/000017_add_teams_and_boards.up.sql

```diff
@@ -11,16 +11,42 @@ ALTER TABLE {{.prefix}}blocks ADD COLUMN board_id VARCHAR(36);
 ALTER TABLE {{.prefix}}blocks_history ADD COLUMN board_id VARCHAR(36);
 
 {{- /* cleanup incorrect data format in column calculations */ -}}
+{{- /* then move from 'board' type to 'view' type*/ -}}
 {{if .mysql}}
-UPDATE {{.prefix}}blocks SET fields = JSON_SET(fields, '$.columnCalculations', cast('{}' as json))  WHERE fields->'$.columnCalculations' = cast('[]' as json);
+UPDATE {{.prefix}}blocks SET fields = JSON_SET(fields, '$.columnCalculations', cast('{}' as json)) WHERE fields->'$.columnCalculations' = cast('[]' as json);
+
+UPDATE {{.prefix}}blocks b
+  JOIN (
+    SELECT id, fields->'$.columnCalculations' as board_calculations from {{.prefix}}blocks
+    WHERE fields -> '$.columnCalculations' <> cast('{}' as json) and fields -> '$.columnCalculations' <> cast('[]' as json)
```

Comment:
nit: we don't need to consider `[]` here as it has just been replaced in the previous `UPDATE`

```suggestion
    WHERE fields -> '$.columnCalculations' <> cast('{}' as json)
```